src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -329,29 +287,26 @@ const waitForContext = (
             if (testId == context?.testId && context?.type == contextType) {
               console.log(
                 Date.now() +
-                  ` Received "${contextType}" (from current context) for test: "${testId}"`
+                ` Received "${contextType}" (from current context) for test: "${testId}"`
               );
               if (executionListener) executionListener.unsubscribe();
               resolve(context);
             } //do not warn as it will be ignoring mismatches which will be common
             else {
               console.log(
                 Date.now() +
-                  ` CHecking for current context of type "${contextType}" for test: "${testId}" Current context did ${
-                    context ? "" : "NOT "
-                  } exist,
-            had testId: "${context?.testId}" (${
-                    testId == context?.testId ? "did match" : "did NOT match"
-                  })
-            and type "${context?.type}" vs ${contextType} (${
-                    context?.type == contextType ? "did match" : "did NOT match"
-                  })`
+                ` CHecking for current context of type "${contextType}" for test: "${testId}" Current context did ${context ? "" : "NOT "
```

Comment:
```suggestion
                ` Checking for current context of type "${contextType}" for test: "${testId}"
            Current context did ${context ? "" : "NOT "
```

Reply:
ok, well I'm not the arbiter of truth on what is good for Finsemble - happy to merge if @kriswest is.